frontend/views/utils/humanDate.js

```diff
@@ -0,0 +1,5 @@
+export function humanDate (datems, opts = {}) {
+  const date = new Date(datems)
+  const locale = navigator.languages ? navigator.languages[0] : navigator.language
+  return date.toLocaleDateString(locale, opts)
```

Comment:
We don't need to fetch the locale each time this function is called, that can be done once at the top of this file.

Then this function can be a one-liner:

```js
return new Date(datems).toLocaleDateString(locale, opts)
```